mne_bids/mne_bids.py

```diff
@@ -253,7 +271,7 @@ def _scans_tsv(raw, raw_fname, fname, verbose):
     """
     # get MEASurement date from the data info
     meas_date = raw.info['meas_date']
-    if isinstance(meas_date, (np.ndarray, list)):
+    if isinstance(meas_date, (np.ndarray, list, tuple)):
         meas_date = meas_date[0]
```

Comment:
In master of MNE-Python, `meas_date` has just been formatted to consistently be a tuple. So soon, we will be able to get rid of np.ndarray and list over there. Should I make a comment so we don't forget? https://github.com/mne-tools/mne-python/pull/5500

Reply:
I would make a comment and add an issue for the mne version that we will freeze for that particular release